### DISCUSSION TITLE
grid path finding using astar

### DISCUSSION THREAD

**Discussion by timi5691** [vlang/v#15099] [General]
Source: https://github.com/timi5691/grid-path-finding-in-v-programing-language
Video: https://youtu.be/f63WGUufQQQ

**Comment by medvednikov** [comment]
Thanks!

It's better to share them via github.

**Comment by Wajinn** [comment]
Great to see you put them on GitHub.  Also, didn't you have or originally show some others?

**Reply by timi5691** [reply]
(sorry for any mistake, i'm not good at english.)
I did have a show. link to the show is the youtube link above.

**Reply by Wajinn** [reply]
I meant that your GitHub doesn't have the other program.  People can get a bit worried about downloading programs from random or unknown websites versus being on GitHub (which is famous), where they can feel or see those programs are safe.
